.azure/infrastructure/test.bicepparam

```diff
@@ -29,9 +29,12 @@ param slackNotifierSku = {
   applicationServicePlanName: 'Y1'
   applicationServicePlanTier: 'Dynamic'
 }
-param postgresSku = {
-  name: 'Standard_B2s'
-  tier: 'Burstable'
+param postgresConfiguration = {
+  sku: {
+    name: 'Standard_B2s'
+    tier: 'Burstable'
+  }
+  enableQueryPerformanceInsight: false
```

Comment:
Kan vi skru på for test og? Vet frontend bruker dette miljøet enda, de opplevd sluggishness der og. (Jeg og @knuhau har repro'et det og)
```suggestion
  enableQueryPerformanceInsight: true
```

Reply:
aha, yesss